data/db-config.js

```diff
@@ -0,0 +1,3 @@
+const knex = require("knex");
+const config = require("../knexfile.js");
+module.exports = knex(config.development);
```

Comment:
You could make this load dynamically by doing something like:
```js
module.exports = knex(config[process.env.NODE_ENV]);
```

Reply:
That is cool, I'll give that a shot for the next one!